docs/install-azure-repos.md

```diff
@@ -0,0 +1,180 @@
+[Go back to the main documentation page](../README.md)
+# Installing Frogbot on Azure Repos repositories
+
+To install Frogbot on Azure Repos repositories:
+
+1. Go to Azure DevOps Pipelines of the relevant project, and add `New pipeline`.
+2. Set `Azure Repos Git` as your code source.
+3. Select the repository you'd like Frogbot to scan.
+4. Select 'Starter Pipeline' and use `frogbot-scan-pr.yml` and `frogbot-scan-and-fix.yml`:
+5. Make sure you have the connection details of your JFrog environment.
```

Comment:
I think this should be the very first step.

Reply:
done